x-pack/plugins/infra/public/components/logging/log_analysis_results/analyze_in_ml_button.tsx

```diff
@@ -62,7 +62,7 @@ const getOverallAnomalyExplorerLink = (pathname: string, jobId: string, timeRang
     },
   });
 
-  const hash = `/explorer?${stringify(urlUtils.encodeQuery({ _g }), { encode: false })}`;
```

Comment:
Removing the `encode: false` here changes the output of these links. `urlUtils.encodeQuery` will counteract over-zealous encoding with `encodeUriQuery`, but then `stringify` readds it. Whilst it's ultimately harmless, the links are nicer imo with the unnecessary encodings replaced.